sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationAsyncClientTest.java

```diff
@@ -539,15 +542,36 @@ public void listWithMultipleKeys(HttpClient httpClient, ConfigurationServiceVers
 
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.data.appconfiguration.TestHelper#getTestParameters")
-    public void listConfigurationSettings(HttpClient httpClient, ConfigurationServiceVersion serviceVersion) {
+    public void listConfigurationSettingsWithNullSelector(HttpClient httpClient, ConfigurationServiceVersion serviceVersion) {
         client = getConfigurationAsyncClient(httpClient, serviceVersion);
+        String key = getKey();
+        String key2 = getKey();
 
-        SettingSelector settingSelector = new SettingSelector();
-        settingSelector.setKeyFilter("prodDBConnection");
-        com.azure.core.util.Context context = new com.azure.core.util.Context(SettingFields.KEY, SettingFields.VALUE);
+        // Delete all existing settings in the resource
+Mono<Void> deleteAllSettingsMono = client.listConfigurationSettings(null)
```

Comment:
nit : `Mono<Void>` is not starting from connect indentation. Is it just git showing this way or this should be fixed ?